server/seeds/demoSeed.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require("mongoose");
+const User = require("../models/User");
+const connectDB = require("../db");
+
+connectDB();
+
+const seedUser = [
+    {
+        name: 'demouser123',
+        email: 'demouser123@gmail.com',
+        password: 'demouserpassword123',
+    },
+];
```

Comment:
And particular reason for making this an array?

Reply:
I changed it to an object